src/components/Photo.jsx

```diff
@@ -1,15 +1,19 @@
 import styles from '../styles/photo'
 
 import React from 'react'
+import { Link } from 'react-router'
 
-const STACK_FILES_DOWNLOAD_PATH = 'http://cozy.local:8080/files/download'
+import { STACK_FILES_DOWNLOAD_PATH } from '../constants/config'
 
-const Photo = ({ photo }) => {
+export const Photo = ({ photo, router }) => {
```

Comment:
@CPatchane yes, React needs the `key`attribute so that list changes can be optimized. And you're right, Preact doesn't need it. But I think it's better to leave it here so that we stay react-compatible.